src/core/main.c

```diff
@@ -667,7 +666,6 @@ static int parse_config_file(void) {
                 { "Manager", "CrashReboot",               config_parse_bool,             0, &arg_crash_reboot                      },
                 { "Manager", "ShowStatus",                config_parse_show_status,      0, &arg_show_status                       },
                 { "Manager", "CPUAffinity",               config_parse_cpu_affinity2,    0, NULL                                   },
-                { "Manager", "JoinControllers",           config_parse_join_controllers, 0, &arg_join_controllers                  },
```

Comment:
I know that it seems that nobody is using this, but I'd prefer to follow the usual protocol, and add a funcition that says "Sorry, this has been removed.". This has been documented for a long time and people might have this configured somewhere, even if it doesn't work properly, and I prefer to be able to say that we gave them a heads-up.